manifest.json

```diff
@@ -431,11 +441,12 @@
         "targetDir": "midi-js-a8a842"
       },
       "ngJoyride": {
-        "version": "5b25d076405c9a787db602d2a8ff49919228f4fc",
-        "downloadFormat": "zip",
-        "url": "https://github.com/seanlip/ng-joyride/archive/5b25d076405c9a787db602d2a8ff49919228f4fc.zip",
-        "rootDirPrefix": "ng-joyride-",
-        "targetDir": "ng-joyride-5b25d0",
+        "version": "",
+        "downloadFormat": "files",
+        "url": "https://raw.githubusercontent.com/NishealJ/ng-joyride/master/",
```

Comment:
@NishealJ, please make changes here: https://github.com/oppia/ng-joyride via a PR and then use this here. Thanks!

Reply:
yes, i've raised a PR when its in i'll add it here